apps/liteprotocoltester/infra.env

```
@@ -0,0 +1,11 @@
+TEST_INTERVAL_MINUTES=10
+START_PUBLISHING_AFTER=120
+NUM_MESSAGES=300
+DELAY_MESSAGES=1000
```

Maybe we could have a more explicit var name;P
```suggestion
MSG_INTERVAL_MILLIS=1000
```

---

Good idea! I need to apply such change more deep. I will check!